packages/cozy-dataproxy-lib/src/search/SearchEngine.ts

```diff
@@ -179,14 +180,13 @@ export class SearchEngine {
     const fieldsToIndex = SEARCH_SCHEMA[doctype]
 
     const flexsearchIndex = new FlexSearch.Document<CozyDoc, true>({
-      tokenize: 'forward',
+      tokenize: 'full',
```

Comment:
nit: this change is not documented in the commit message and is not part of the "do not use flexsearch store" scope. Maybe you can add some precision in the commit message or extract it in another commit?

Reply:
Oh my, it's a mistake, thank you!